tests/setup/kfctl_openshift.yaml

```diff
@@ -169,9 +169,19 @@ spec:
         name: manifests
         path: model-mesh
     name: model-mesh
+  - kustomizeConfig:
+      overlays:
+        - metadata-store-mysql
+        - ml-pipeline-ui
```

Comment:
I don't think we should discuss if we want to test `ml-pipeline-ui` overlay here. As it is just basic tests, we should test basic flows, and the UI seems to me like a special case.